pyvcloud/vcd/vdc.py

```diff
@@ -340,6 +342,21 @@ def list_resources(self, entity_type=None):
                     result.append({'name': vapp.get('name')})
         return result
 
+    def list_edgegateways(self):
```

Comment:
nit : list_edge_gateways

Reply:
method naming is fixed in latest commit fixed in latest commit 
get the edge gateway list on the link is fixed in the latest commit

I had a problem to find the Link `rel="fullEdgeGateway"`, couldn't see it on my test vdc and I also couldn't find it in the docs https://pubs.vmware.com/vcd-820/index.jsp?topic=%2Fcom.vmware.vcloud.api.sp.doc_27_0%2FGUID-EECCA924-4DCE-490F-B352-5C212AD22411.html&resultof=%22%65%64%67%65%47%61%74%65%77%61%79%73%22%20%22%65%64%67%65%67%61%74%65%77%61%69%22%20%22%4c%69%6e%6b%22%20%22%6c%69%6e%6b%22%20